composer.json

```diff
@@ -28,10 +28,10 @@
         "drupal/core": "8.7.0",
         "drupal/devel": "^2.1",
         "drupal/easy_breadcrumb": "^1.12",
+        "drupal/honeypot": "^1.30",
         "drupal/maxlength": "^1.0@beta",
         "drupal/paragraphs": "^1.10",
         "drupal/pathauto": "^1.6",
-        "drupal/recaptcha": "^2.4",
```

Comment:
Can you please properly delete recaptcha from the site, removing it from the composer.json file is not enough

Reply:
I used ```drush pm:uninstall``` and ```composer remove```. Devastated that you would think otherwise.

Reply:
the files weren't in the pull request, sooooooo...